packages/xdl/src/ApiV2.ts

```diff
@@ -165,7 +157,7 @@ export default class ApiV2Client {
       reqOptions.data = options.body;
     }
 
-    reqOptions = _.merge({}, reqOptions, extraRequestOptions);
+    reqOptions = merge({}, reqOptions, extraRequestOptions);
```

Comment:
I thought lodash merge might have been used in this code before because it does deep merge? Request options include some nested objects, e.g. `headers`.

Reply:
Oh, hmm.. in that case let's keep the old `merge` code.